components/style/properties/data.py

```diff
@@ -212,16 +213,24 @@ def experimental(self, product):
 
     # FIXME(emilio): Shorthand and Longhand should really share a base class.
     def explicitly_enabled_in_ua_sheets(self):
-        return self.internal
+        if self.enabled_in in ["ua", "chrome"]:
+            return True
+        assert self.enabled_in in ["", "content"]
+        return False
 
-    # TODO(emilio): Change the `internal` field to something like `enabled_in`.
     def explicitly_enabled_in_chrome(self):
+        if self.enabled_in == "chrome":
+            return True
+        assert self.enabled_in in ["", "ua", "content"]
         return False
 
+    def enabled_in_content(self):
```

Comment:
Maybe something like `not_explicitly_enabled` can be less confusing? Since an experimental property can also be enabled in content if its pref is on.

Reply:
I think this would also be slightly confusing... I think I'll stick with the name for now, I think.